app/services/katello/pulp3/repository_mirror.rb

```diff
@@ -75,7 +75,7 @@ def fetch_repository
       end
 
       def version_href
-        fetch_repository.latest_version_href
+        fetch_repository&.latest_version_href
```

Comment:
Even with this, `latest_content_counts` tries to do `api.repository_versions_api.read(version_href)` which errors out. We could just skip that repository perhaps if version_href is `nil`.

Reply:
To reproduce:

1) Create a CVV assigned to some env with some yum content
2) Sync that CVV to the smart proxy
3) Promote the CVV to a new LCE
4) Add that LCE to the smart proxy
5) Run the content counts update
6) See the error:
```
Exception:

ArgumentError: Missing the required parameter 'rpm_rpm_repository_version_href' when calling RepositoriesRpmVersionsApi.read

Backtrace:

/home/vagrant/foreman/.vendor/ruby/2.7.0/gems/pulp_rpm_client-3.22.5/lib/pulp_rpm_client/api/repositories_rpm_versions_api.rb:238:in `read_with_http_info'
/home/vagrant/foreman/.vendor/ruby/2.7.0/gems/pulp_rpm_client-3.22.5/lib/pulp_rpm_client/api/repositories_rpm_versions_api.rb:221:in `read'
/home/vagrant/katello/app/services/katello/pulp3/repository_mirror.rb:228:in `latest_content_counts'
/home/vagrant/katello/app/models/katello/concerns/smart_proxy_extensions.rb:133:in `block in update_content_counts!'
/home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activerecord-6.1.7.6/lib/active_record/relation/delegation.rb:88:in `each'
/home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activerecord-6.1.7.6/lib/active_record/relation/delegation.rb:88:in `each'
/home/vagrant/katello/app/models/katello/concerns/smart_proxy_extensions.rb:131:in `update_content_counts!'
/home/vagrant/katello/app/lib/actions/katello/capsule_content/update_content_counts.rb:15:in `run'
```

Reply:
Updated to skip repos not synced to capsule, i.e version_href = nil